### ISSUE TITLE
Groups can't be seen or accessed on album pages in compact UI

### ISSUE THREAD

**Issue by towerofnix** [hsmusic/hsmusic-wiki#45]
This is pretty bad!

Maybe a small bar across the top of the screen (beneath the banner) with the names of the groups would work? I don't really feel as though this info fits within the main content of the page, but it should be accessible at all screen dimensions.

(13 Mar 2022)

**Comment by towerofnix** [comment]
Done in dacdf44c64688ea56bceb001a5ef431c1905061f!
